src/public/js/new_product_validation.js

```diff
@@ -0,0 +1,99 @@
+const submit_bt = document.getElementById("submit-btn");
+const pname = document.getElementById("name");
+const cpu = document.getElementById("cpu");
+const memory = document.getElementById("memory");
+const graphics = document.getElementById("graphics");
+const details = document.getElementById("details");
+const price = document.getElementById("price");
+const formerror = document.getElementById("formerror");
+const main = document.getElementById("main");
+
+// TODO: Dodać obsluge zdjec
+
+function validate() {
+  if (pname.value === '' || pname.value.length > 500) {
+    pname.value = '';
+    return ("Real name is required");
+  }
+  if (cpu.value === '' || cpu.value.length > 500) {
+    cpu.value = '';
+    return ("Correct cpu is required");
+  }
+  if (memory.value === '' || /\d+(gb|Gb|GB)?/.test(memory.value) === false) {
```

Comment:
Przechodzi input `8` oraz `8GB` i potem w takiej postaci zapisuje się w BD. Może na backendzie zróbmy jakiegoś trima, żeby zapisywać samą liczbę?

Reply:
Tak było do tej pory w mock db, myślę że możemy to poprawić jak ustalimy to konsekwentnie, bo wtedy trzeba też wszystkie widoki zmienić